public/app/features/alerting/unified/components/rules/EditRuleGroupModal.tsx

```diff
@@ -192,7 +193,7 @@ export const RulesForGroupTable = ({
 
   return (
     <div className={styles.tableWrapper}>
-      <DynamicTable items={rows} cols={columns} />
+      <DynamicTable items={rows} cols={columns} pagination={{ itemsPerPage: DEFAULT_PER_PAGE_PAGINATION / 4 }} />
```

Comment:
I think it's best to just use a fixed number here to determine how many we want to fit in the modal. 

If `DEFAULT_PER_PAGE_PAGINATION` increases to some larger value it might affect the scrolling of the modal content. Inversely if it becomes smaller you might not have many items to view. And I'm not sure what happens if `DEFAULT_PER_PAGE_PAGINATION / 4` is a float / fractal 😅

Reply:
💯